pos_require_product_quantity/static/src/js/pos.js

```diff
@@ -0,0 +1,71 @@
+/*
+  Copyright 2019 Coop IT Easy SCRLfs
+  	    Robin Keunen <robin@coopiteasy.be>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define(
+
+'pos_require_product_quantity.pos_require_product_quantity',
+function (require) {
+"use strict";
+
+var core = require('web.core');
+var models = require('point_of_sale.models');
+var screens = require("point_of_sale.screens");
+
+var _t = core._t;
+var orderline_prototype = models.Orderline.prototype;
+
+models.Orderline = models.Orderline.extend({
+    initialize: function (attr, options) {
+        orderline_prototype.initialize.call(this, attr, options);
+
+        var unit = this.get_unit();
+        if (unit) {
+            if (unit.category_id[1] === 'Unit') {
```

Comment:
what is unit.category_id[1] ? the name of the category ? if yes, it will not work in non english context ? or did I missed something ?